features/employment/statutory_pay.feature

```diff
@@ -0,0 +1,25 @@
+Feature:
+    "Statutory pay"
+
+    Scenario: The client is receiving statutory sick pay only
+        Given I am using version 5 of the API
+        And I create an assessment with the following details:
+            | client_reference_id | NP-FULL-1  |
+            | submission_date     | 2021-05-10 |
```

Comment:
Think this date should be closer to the present day than some date in 2021? I just had to nudge another test due to this same date